insights/parsers/du.py

```diff
@@ -0,0 +1,59 @@
+"""
+du - Command
+============
+
+Module for the processing of output from the ``du`` command stored in
+file.  The base class ``DiskUsage`` provides all of the functionality.
+
+This example demonstrates the funcionality of ``DiskUsage`` class.
+
+Examples:
+    >>> FILE_CONTENT = '''
+    ... COMMAND> du -sh /var/lib/mongodb
+    ...
+    ... 22G     /var/lib/mongodb
```

Comment:
I think it's a bad idea to use 'human' units for this command - it should be in kilobytes (`du -sk`).  You retain a lot more accuracy, and it's much easier for rules based on this input to compare values to other things - e.g. free disk space, file sizes, etc.